### PR TITLE
[rc] Add test fixtures

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Partially resolves issue: https://github.com/blackbaud/skyux-lib-testing/issues/26

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-rc-add-test-fixtures](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-rc-add-test-fixtures)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-rc-add-test-fixtures](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-rc-add-test-fixtures)

File: src/app/public/testing/alert-fixture.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    fixture: ComponentFixture<any>,
+    skyTestId: string
+  ) {
+    this.debugEl = SkyAppTestUtility.getDebugElementByTestId(fixture, skyTestId, 'sky-alert');
+  }
+
+  /**
+   * Closes the alert.  If the alert is not closeable, an error is thrown.
+   */
+  public close() {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/testing/alert-fixture.ts:84]: Needs a return type.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-rc-add-test-fixtures](https://host.nxt.blackbaud.com/skyux-indicators?_sp.spa=pr-rc-add-test-fixtures)

**Pull request merged**